tests/jsx/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,5 +1,84 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`conditional-expression.js 1`] = `
+<div>
+  {isCool ? "cool" : "very nice"}
+</div>;
+
+// should get formatted JSX style
+const a = something ? "string" : <div />;
+
+// should not get formatted JSX style
+const a = something ? "string" : "other string";
+
+<div>
+  {reallyLongChainOfConditionalExpressions ? (
+    "why"
+  ) : yes ? (
+    "it"
+  ) : is ? (
+    "!"
+  ) : (
+    ":)"
+  )}
+</div>;
+
+<div>
+  {reallyLongChainOfConditionalExpressions ? (
+    <span>why</span>
+  ) : yes ? (
+    <span>it</span>
+  ) : is ? (
+    <span>!</span>
+  ) : (
+    <span>:)</span>
+  )}
+</div>;
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<div>
+  {isCool ? (
```

Comment:
Makes sense; I'll take a look at `printMemberChain`; sounds like exactly the pattern I need.